rest-api/src/main/java/com/cloudbees/workflow/rest/external/FlowNodeExt.java

```diff
@@ -105,6 +105,15 @@ public ErrorExt getError() {
         return error;
     }
 
+    public void setParameterDescription(String desc) {
+        this.parameterDescription = desc;
+    }
+
+    @JsonInclude(JsonInclude.Include.NON_NULL)
+    public String getParameterDescription() {
```

Comment:
Would be slightly less confused if plural was used, e.g. `getParametersDescription`